README.md

```diff
@@ -144,15 +144,54 @@ We recommend using the JAR instead of the WAR file deployment, because it's much
 Note that with the 1.4 release the tenants database pool configuration changed from Tomcat DBCP in XML to an embedded Hikari, configured by environment variables, see above.
 
 
-INSTRUCTIONS: How to execute Integration Tests
+INSTRUCTIONS: How to run tests
 ============
-> Note that if this is the first time to access MySQL DB, then you may need to reset your password.
 
-Run the following commands:
```

Comment:
Should we really remove the 'how to execute integration tests` part?

Reply:
I'm re-titling this section because it is now about all tests: "Unit" and "Integration" (see lines 150 and 162).

We might also want to call out "e2e" or "acceptance" or "smoke" or "bdd/cucumber" tests separately, but my idea for now is just to differentiate "quick/independent/Unit" and "slow/external dependencies/Integration". I care less about abusing those terms than making it easy and possible for new devs to actually run tests reliably. I think we also need to clean up, simplify, and otherwise improve the gradle tasks because they are just bananas right now.